packages/web-worker/README.md

```diff
@@ -0,0 +1,53 @@
+# @bugsnag/web-worker
+
+This package contains the web worker / service worker implementation of the Bugsnag library.
+
+**This package is still in the beta phase, so should be used with caution in production apps.**
+
+We would welcome feedback on this package via GitHub or directly to our [Support](mailto:support@bugsnag.com) team.
+
+## Features and limitations
+
+This early release offers basic functionality for web workers and service workers, with the intent to support Chrome extension development using manifest v3. Using this library, you will be able to:
+
+- notify errors from within service workers and web workers, including browser extensions
+- detect and automatically notify unhandled errors from service workers and web workers, excluding Chrome browser extensions
+
+Due to limitations in the Chrome Runtime API, it is not possible to catch unhandled errors so you will need to catch these from within your code and notify these errors manually.
```

Comment:
Is this paragraph pertaining to Chrome browser extensions?  If so I guess it's redundant?

Reply:
correct, `chrome.runtime` is used in service workers hosting background tasks for browser extensions, and we are unable to catch unhandled errors here. perhaps we could specify `chrome.runtime` in the above "excluding Chrome browser extensions" ?

Reply:
I think we should either 

1. just remove this paragraph entirely
2. ensure that the link is made explicit (e.g. by saying "Due to limitations in the Chrome Runtime API, which is used for service workers hosting background tasks in Chrome browser extensions")
3. remove this paragraph, but append "due to limitations  in the Chrome Runtime API" to the bullet point

Reply:
option 3 sounds clean to me. happy @tomlongridge ?

Reply:
Agree